package/CHANGELOG

```diff
@@ -13,15 +13,26 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/17 utkbansal, kain88-de
+
+mm/dd/17 utkbansal, kain88-de, xiki-tempula, kaplajon
+
 
   * 0.16.1
 
 Enhancements
   * Universe now works with StringIO objects for topologies and trajectories.
+  * Residues with the same residue ids are not merged by default now 
```

Comment:
There are lots of pieces that should not show up in your PR, they come from elsewhere. Something went wrong when you merged. Rebase against develop.